packages/plugin-eslint/vite.config.integration.ts

```diff
@@ -5,6 +5,8 @@ import { tsconfigPathAliases } from '../../tools/vitest-tsconfig-path-aliases.js
 export default defineConfig({
   cacheDir: '../../node_modules/.vite/plugin-eslint',
   test: {
+    // eslint-disable-next-line @typescript-eslint/no-magic-numbers
```

Comment:
unfortunately as I said on standup, that is a broken `nx graph` on windows due to nested `.git`, it will never construct so no amount of timeout will fix the test, I have a fix in logs branch that will hopefully be merged soon

Reply:
removed.